.travis.yml

```diff
@@ -28,10 +26,10 @@ addons:
         key_url: 'https://apt.llvm.org/llvm-snapshot.gpg.key'
       - ubuntu-toolchain-r-test
     packages:
-      - llvm-10-tools
-      - llvm-10-dev
-      - libclang-10-dev
-      - lldb-10
+      - llvm-11-tools
```

Comment:
Can we reuse approach used in the translator - https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/33d49464beee75e02529658ea6682eec15a1cf93/.travis.yml#L29-L33

Reply:
Done